src/game/MotionGenerators/MotionMaster.h

```diff
@@ -74,8 +75,39 @@ enum MovementGeneratorType
     EXTERNAL_WAYPOINT_MOVE          = 17,                   // Only used in UnitAI::MovementInform when a waypoint is reached. The pathId >= 0 is added as additonal value
     EXTERNAL_WAYPOINT_MOVE_START    = 18,                   // Only used in UnitAI::MovementInform when a waypoint is started. The pathId >= 0 is added as additional value
     EXTERNAL_WAYPOINT_FINISHED_LAST = 19,                   // Only used in UnitAI::MovementInform when the waittime of the last wp is finished The pathId >= 0 is added as additional value
+
+    FORMATION_MOTION_TYPE           = 20,                   // TargetedMovementGenerator.h
```

Comment:
How will it work in creature table, in DB it will be still 2?

Reply:
That have nothing to do with defaut movement type in db.
Its only for movement generator internally and used for follower i should rename it.